guides/it/linux_av.md

```diff
@@ -81,50 +113,25 @@ You can test the configuration of your AV software using the test files provided
 #### ClamAV
 [ClamAV](https://www.clamav.net/) is an open source AV tool that is popular on Linux - it can be configured in a way that meets the requirements above, but this involves enabling on-access scanning.
 
-[Installation and configuration instructions](linux_av.md)
-
-Unfortunately the version in the Ubuntu repositories isn't kept up to date with upstream releases as fast as we need it to be so the current recommendation is to compile from source:
-
-[ClamAV compilation guide](https://docs.clamav.net/manual/Installing/Installing-from-source-Unix.html)
-
-You may need to replace
-```
-python3 -m pip install --user cmake pytest
-```
-
-with
-
-```
-sudo python3 -m pip install cmake
+**Debian/Ubuntu:**
+```bash
+sudo apt install clamav clamav-daemon
 ```
 
 Once installed enable and start the systemd service:
 
-```
+```bash
 sudo systemctl enable clamav-daemon
 sudo systemctl start clamav-daemon
 ```
 
-We also want to make a quarantine directory:
-```
-sudo mkdir /root/quarantine
-```
-
-You can check the installed version using:
-
-```
-clamscan --version
-```
-
-and scan using:
-
-```
-clamscan
+We also want to make a quarantine directory that requires elevated privileges to access:
```

Comment:
Actually isn't that sort of reintroducing the problem, the quarantining is meant to fix? Your regular user now still has access to that file without needing to `sudo`?

Reply:
No, you still need to `sudo`, because you are not `clamav` and you are not in the `clamav` group ; you don't own this  folder so you need some kind of elevation to look in there.

None of the Linux workstation users are expected to be "regular" users TBH.